alerting/src/main/kotlin/org/opensearch/alerting/resthandler/RestGetFindingsAction.kt

```diff
@@ -59,9 +57,7 @@ class RestGetFindingsAction : BaseRestHandler() {
 
         val getFindingsSearchRequest = GetFindingsRequest(
             findingID,
-            table,
-            severity,
```

Comment:
why are we removing severity??

Reply:
we don't need to change the behavior of existing Alerting Plugin [API](https://opensearch.org/docs/latest/observing-your-data/alerting/api/#search-the-findings-index)]. This was the part of prev commit which was merged.